src/git.ts

```diff
@@ -472,6 +473,22 @@ function getRevisionIdFromObject(object: string): string {
   return git(['rev-list', '-n', '1', object]);
 }
 
+function getCWD(module?) {
```

Comment:
You might wanna look at `renderer/index.ts#resolvePath()` or `utils.ts#cwd()`. Is this what you had in mind?

Reply:
I just figured I had to use this, and abstracted a piece of existing code into a function. I can certainly try, and perhaps replace the other call as well.